src/orchestratord/src/bin/orchestratord.rs

```diff
@@ -440,6 +435,67 @@ async fn run(args: Args) -> Result<(), anyhow::Error> {
         .run(),
     );
 
+    mz_ore::task::spawn(
+        || "console controller",
+        k8s_controller::Controller::namespaced_all(
+            client.clone(),
+            controller::console::Context::new(
+                controller::console::Config {
+                    enable_security_context: args.enable_security_context,
+                    enable_prometheus_scrape_annotations: args.enable_prometheus_scrape_annotations,
+                    image_pull_policy: args.image_pull_policy,
+                    scheduler_name: args.scheduler_name,
+                    console_node_selector: args.console_node_selector,
+                    console_affinity: args.console_affinity,
+                    console_tolerations: args.console_tolerations,
+                    console_default_resources: args.console_default_resources,
+                    network_policies_ingress_enabled: args.network_policies_ingress_enabled,
+                    network_policies_ingress_cidrs: args.network_policies_ingress_cidrs,
+                    default_certificate_specs: args.default_certificate_specs,
+                    console_http_port: args.console_http_port,
+                    balancerd_http_port: args.balancerd_http_port,
+                },
+                client.clone(),
+            )
+            .await,
+            watcher::Config::default().timeout(29),
+        )
+        .with_controller(|controller| {
+            controller
+                .owns(
+                    Api::<Deployment>::all(client.clone()),
+                    watcher::Config::default()
+                        .labels("materialize.cloud/mz-resource-id")
+                        .timeout(29),
+                )
+                .owns(
+                    Api::<Service>::all(client.clone()),
+                    watcher::Config::default()
+                        .labels("materialize.cloud/mz-resource-id")
+                        .timeout(29),
+                )
+                .owns(
+                    Api::<Certificate>::all(client.clone()),
+                    watcher::Config::default()
+                        .labels("materialize.cloud/mz-resource-id")
+                        .timeout(29),
+                )
+                .owns(
+                    Api::<NetworkPolicy>::all(client.clone()),
+                    watcher::Config::default()
+                        .labels("materialize.cloud/mz-resource-id")
+                        .timeout(29),
+                )
+                .owns(
+                    Api::<ConfigMap>::all(client.clone()),
+                    watcher::Config::default()
+                        .labels("materialize.cloud/mz-resource-id")
```

Comment:
Does this add ownership for all objects with any resource ID? Won't this controller then own objects created by the Balancerd and Materialize controllers?